Fix Build and Update Author Kevin (me)

PsychoData opened the pull request:
Add LinkedIn

PsychoData reviewed (commented):
error in https://github.com/SysManSquad/sysmansquadblogdev/pull/56 about `executing "partials/_funcs/get-page-images.html" at <$imgParams>: range can't iterate over S`

Seems this was caused by Params.images not being an array to iterate through. 

(Needed to trigger a fresh workflow run with https://github.com/SysManSquad/sysmansquadblogdev/pull/56 to make certain the issue I was seeing locally would also show in the WorkFlow run, but this was the fix locally)

PsychoData commented:
(Although, Hugo does still seem to complain if I compile it with Drafts / `-D`, but I can just create Local "Live" pages to test with) 
![image](https://github.com/SysManSquad/sysmansquadblogdev/assets/3719116/62d9762e-dd15-4618-9e10-d7d1bb0559aa)

aaroneg reviewed (changes requested):
Fix the line endings/other whitespace problems so this is actually a reviewable pr.

PsychoData commented:
I added VSCode settings for the Line Endings to be maintained as LF only, and fixed them back 😆 
The latest commit compare shows the WHOLE file changing again, but the OVERALL files changed graph on GIthub shows the ACTUAL only a few characters here and there changing
